modules/tools/velero/velero-values.tpl

```diff
@@ -0,0 +1,35 @@
+configuration:
+  provider: aws
+
+  backupStorageLocation:
+    name: default
+    bucket: ${bucket_name}
+    prefix: ${cluster_name}
+    config:
+      region: ${region}
+
+  volumeSnapshotLocation:
+    name: aws-s3
+    config:
+      region: ${region}
+    
+initContainers:
+   - name: velero-plugin-for-aws
+     image: velero/velero-plugin-for-aws:v1.2.0
+     imagePullPolicy: IfNotPresent
+     volumeMounts:
+       - mountPath: /target
+         name: plugins
+  
+credentials:
+  secretContents:
+    cloud: |
+      [default]
+      aws_access_key_id=${velero_accesskey_id}
+      aws_secret_access_key=${velero_accesskey_secret}
+
+serviceAccount:
+  server:
+    create: true
+    name: velero
+    annotations:
```

Comment:
we don't have to do this now, but we should be using EKS IAM roles for service accounts (https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) to provide AWS creds to workloads running within EKS (assuming velero supports it).  we currently do this with cert-manager and external-dns, although we don't do it for Vault because they didn't support IRSA at the time.

Reply:
I got this working but only after adding "fsGroup: 65534" to the security context in values for the Velero helm release. There is an issue (https://github.com/vmware-tanzu/velero/issues/2981) that apparently is resolved in K8s 1.19 (https://github.com/kubernetes/kubernetes/issues/82573). I’ve removed the user, access key, etc..that I previously created.